GiniVision/Classes/AnalysisViewController.swift

```diff
@@ -23,6 +23,15 @@ import UIKit
      - parameter action:  The action to be performed after the user tapped the error view.
      */
     func displayError(withMessage message: String?, andAction action: NoticeAction?)
+    
+    /**
+     In case that the `GiniVisionDocument` analysed is an image it will display a no results screen
+     with some capture suggestions. It won't show any screen if it is not an image, pointing out
+     on the completion handler that any screen was shown.
+     
+     - parameter completion: Handler that returns `true` if a screen was shown or `false` if it wasn't.
+     */
+    func displayNoResultsScreen(completion: ((Bool) -> ()))
```

Comment:
I really dislike this. I method called `displayNoResultsScreen` should, in fact, display the screen.

Right now, it says "If you tell me to display something, I'll give you back a result saying if I decided to listen to you, or not". It's an OOP principles to **tell** objects what to do, not ask them.

Reply:
Yes, probably the name is wrong, but I think it is the only way to proceed. Do you suggest a better name?

Reply:
No name will fix this issue. But maybe the name should reflect that the call might fail. Like `tryDisplayNoResultsScreen`.